src/Documentation/UserManual/DeviceOptiTrack.dox

```diff
@@ -45,7 +45,7 @@
 - \xmlAtt \ref ToolReferenceFrame \OptionalAtt{Tracker}
 
 - \xmlAtt \b ProjectFile The Motive project file (.TTP) to load. \RequiredAtt
-- \xmlAtt \b RunMotiveInBackground If true, will run Motive in the background. If false, you must run Motive on the same PC as PLUS is running on. \RequiredAtt
+- \xmlAtt \b ShowMotiveGUI Set to TRUE if you intend to run the Motive GUI application on the PLUS computer. If set to FALSE, PLUS will internally handle the Motive functionality. \RequiredAtt

```

Comment:
I think this would be more clear:

Set to TRUE to make Plus connect to an already running instance of Motive application (on the same computer where Plus is running). If set to FALSE, PLUS will internally start Motive application, without its graphical user interface hidden.